src/EnergyPlus/GroundHeatExchangers.cc

```diff
@@ -946,13 +972,53 @@ Real64 GLHEVert::doubleIntegral(std::shared_ptr<GLHEVertSingle> const &bh_i, std
 
 //******************************************************************************
 
+void GLHEVert::calcLongTimestepGFunctions(EnergyPlusData &state)
+{
+    switch (this->gFuncCalcMethod) {
+    case GFuncCalcMethod::UniformHeatFlux:
+        this->calcUniformHeatFluxGFunctions(state);
+        break;
+    case GFuncCalcMethod::UniformBoreholeWallTemp:
+        this->calcUniformBHWallTempGFunctions();
+        break;
+    default:
+        assert(false);
+    }
+}
+
```

Comment:
The UBHWTcalc makes use of the same borehole locations and time steps as UHFcalc would, they just need stored in a vector first.